firefox-ios/base-fingerprinting-track.json

```diff
@@ -0,0 +1,106 @@
+[
```

Comment:
@englehardt, should I use a more broader name for the folder like `normalized-lists` instead?

Reply:
Yes let's do that -- in case we have other mobile products using the lists in the future.